packaging/neo4j-desktop/src/main/java/org/neo4j/desktop/runtime/DatabaseActions.java

```diff
@@ -105,16 +97,21 @@ public void stop()
         }
     }
 
-    public void shutdown()
+    private boolean isRunning()
     {
-        if ( isRunning() )
-        {
-            stop();
-        }
+        return server != null;
     }
 
-    public boolean isRunning()
+    private Set<Class> extractCauseTypes( Throwable e )
```

Comment:
This method is used to obtain set of causes and then check if it contains some particular types. This is probably simpler to do with `Exceptions#contains(Throwable, Class...)` method.